src/router/index.js

```diff
@@ -226,6 +232,12 @@ const routes = [
     component: () => import('../pages/ScoreReport.vue'),
     meta: { pageTitle: 'View Scores', requireAdmin: true },
   },
+  {
```

Comment:
Add route guard for LEVANTE and to block student from taking survey before completing rest of tasks